Dockerfile

```diff
@@ -5,6 +5,9 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
 # Useful tools for debugging
 RUN apk add --no-cache jq curl
+RUN apk update
+RUN apk add --upgrade libgcc
+RUN apk add --upgrade libstdc++
```

Comment:
Thank you for the PR:

Can you try bumping the base image to `node:lts-alpine3.12` first, and as a fallback to doing that, do a section commented to be about security patches, where we execute a single `RUN` to `apk update && apk add --upgrade <pageage 1> <package 2> ...`.

Reply:
@consideRatio Thanks for pointing that out. Updating alpine seems to have fixed the vulnerabilities:

```
2020-10-19T11:31:22.289-0500	[34mINFO[0m	Detecting Alpine vulnerabilities...
2020-10-19T11:31:22.297-0500	[34mINFO[0m	Detecting nodejs vulnerabilities...

annas-cool-image:latestt (alpine 3.12.0)
========================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


srv/configurable-http-proxy/package-lock.json
=============================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```